Servers/Classic/npc/custom/eAAC_Scripts/kafraExpress/config.txt
//===== eAthena Script ======================================= 
//= Kafra Express - Config Module
//===== By: ================================================== 
//= Skotlex 
//===== Current Version: ===================================== 
//= 2.4
//===== Compatible With: ===================================== 
//= Any eAthena Version with function support
//===== Description: ========================================= 
//= Part of the Kafra Express Script Package.
//= This module contains the configuration of all modules. 
//===== Additional Comments: ================================= 
//= See <http://www.eathena.deltaanime.net/board/index.php?showtopic=20290>
//= for indepth explanation of the Kafra Express Script Package.
//============================================================ 

-	script	keConfig	-1,{
	//-------------------------------------------------------------------------------
	//Config for the Main Module
	//-------------------------------------------------------------------------------
OnInit:
	set $@ke_saveOnSpot,0;	//If 1, the respawn point is saved where the char is standing.
	set $@keh_hpCost,10;	//Cost of HP healing (per 10HP)
	set $@keh_spCost,100;	//Cost of SP healing (per 10SP)
	set $@kes_cost,50;	//Cost of opening the storage
	set $@kegs_cost,500;	//Cost of opening the guild storage

	//Kafra Pass Options:
	set $@kekp_reset,1; //How many uses a kafra pass has before expiring?
		//(if 0, duration is as long as a temporary variable)
	set $@kekp_reserveCost,100;	//Cost of each Reserve Point in Zeny (0 disable)
	set $@kekp_minReserve,1;	//Min Reserve Points you can earn in a single transaction
	set $@kekp_maxReserve,150;	//Max Reserve Points you can earn in a single transaction

	//Display Options
	set $@ked_users$,"people"; //String used when referring to people (player count).
		// Values like "ppl", "users" or "on" are also recommended.
	end;
	//-------------------------------------------------------------------------------
	//Config for the Bank Module
	//-------------------------------------------------------------------------------
OnLoadBank:
	set $@kebk_depositCost,0;	//Fee for bank deposits in % (10 = 10%)
	set $@kebk_withdrawCost,5;	//Fee for bank withdrawals in % (10 = 10%)
	set $@kebk_minTransact,1000;	//Value of the Minimum Transaction (deposit/withdraw)
	set $@kebk_maxTransact,999999;	//Value of the Maximum Transaction (deposit/withdraw)
	set $@kebk_capacity,99999999;	//Account capacity: Max zeny that can be held.
	set $@kebk_dayInterest,10;	//Savings daily interests in 0.01% units (100 = 1%)
	set $@kebk_monMaintenance,1000;	//Monthly flat fee charged for account maintenance
	set $@kebk_useGlobalBank,0;	//When 1, all charged fees go to $ke_globalbank 
	end;
	//-------------------------------------------------------------------------------
	//Config for the Broadcast Module
	//-------------------------------------------------------------------------------
OnLoadBroadcast:
	//Following costs are for current map costs:
	set $@kebc_showOnline,0;	//if 1, count of recipients will be listed.
	set $@kebc_partyCost,2500;	//party request broadcast cost
	set $@kebc_pvpCost,5000;	//pvp request broadcast cost
	set $@kebc_cost,10000;	//general broadcast cost
	set $@kebc_anonCost,100000;	//anonymous broadcast cost
	set $@kebc_globalFactor,1000;	//When doing server bcs, the prices are increased by this amount (500 = 500% = 5x)
	set $@kebc_discount,15;	//% Discount to apply when the KafraPass is active
	end;
	//-------------------------------------------------------------------------------
	//Config for the Dye and Stylist Module
	//-------------------------------------------------------------------------------
OnLoadDye:
	set $@kedy_enableHairstyle,0;	//Set to 1 to enable haircut style change.
	set $@kedy_styles,23;	//Highest available haircut style (default 23)
	set $@kedy_hair,8;	//Highest available hair dye (RO default is 8)
	
	//Highest available cloth dye depending on job-class
	set $@kedy_clothJN,77;	//Novice (+High/Baby)
	set $@kedy_clothJ1ST,77;	//1st Jobs (+High/Baby)
	set $@kedy_clothJ2ND,77;	//2nd Jobs (+High/Baby)
	set $@kedy_clothJSN,77;	//Super Novice (+Baby)
	end;
	//-------------------------------------------------------------------------------
	//Config for the Job Changer Module
	//-------------------------------------------------------------------------------
OnLoadJobChange:
	set $@kejc_skillsPolicy,0;	//Remaining Skill Points Policy:
		//0: Do not allow job changing until points are used up
		//1: Discard the remaining skill points
		//2: Let extra skill points be carried to next job

	set $@kejc_upperPolicy,0;	//Upper Job management policy:
		//0: Ignore Mode: People can change in any order
		//1: Force Mode: Job changes are forced on the correct path.
	
	set $@kejc_announce,1;	//Do a global announce when someone changes job.
	set $@kejc_resetDye,1;	//Set to 1 to reset cloth dye upon change
	set $@kejc_skipNovice,0;	//0: Novice needs Job 10 to change, 1: Can Skip Novice Class
	set $@kejc_baseSN,30;	//Base lv required to become a Super Novice

	set $@kejc_disable,0;	//Disable certain jobs from the changer:
		//1: Super Novice
		//2: Taekwon/Star Gladiator/Soul Linker
		//4: GunSlinger
		//8: Ninja

	//Note that the change costs can be converted into rewards by specifying a negative amount.
	set $@kejc_base2ND,1;	//Base lv required to change into a second class
	set $@kejc_job2ND,40;	//Job Lv required to change into a second Class
	set $@kejc_cost2ND,-25000;	//Zeny cost when changing into a second Class

	set $@kejc_baseRebirth,99;	//Base lv required to do the Rebirth 
	set $@kejc_jobRebirth,50;	//Job Lv required to do the Rebirth
	set $@kejc_costRebirth,1000000;	//Zeny cost required to do the Rebirth

	set $@kejc_rebirthReset,1;	//When changing to a high novice, reset base level to 1?
	set $@kejc_weaponPolicy,1;	//Set to 1 to give a weapon upon job change
	
	if ($@kejc_weaponPolicy == 0) //don't touch
		end;

	set $@kejc_wBonusLv,50;	//Job level for upgraded weapon version, use 0 to disable
	
	//Weapons awarded to each job upon change (use 0 to disable for a particular class)
	//First classes:
	setarray $@kejc_weapon1[0],
		1501,	//Acolyte/Club
		1701,	//Archer/Bow
		1601,	//Mage/Rod
		1301,	//Merchant/Axe
		1101,	//Swordman/Sword
		1201,	//Thief/Knife
		0,		//Taekwon/Nothing (they don't wield weapons)
		1202,	//Super Novice/Knife
		13010, //GunSlinger/Six Shooter
		1201; //Ninja/Knife
	
	//Normal Weapons for Second classes
	setarray $@kejc_weapon_21[0],
		1551,	//Priest/Bible
		1704,	//Hunter/Composite Bow
		1602,	//Wizard/Rod
		1351,	//Blacksmith/Battle Axe
		1116,	//Knight/Katana
		1250,	//Assassin/Jur
		1550;	//Star Gladiator/Book

	setarray $@kejc_weapon_22[0],
		1801,	//Monk/Waghnak
		1704,	//Bard/Dancer/Composite Bow
		1550,	//Sage/Book
		1351,	//Alchemist/Battle Axe
		1107,	//Crusader/Blade
		1210,	//Rogue/Cutter
		1602;	//Soul Linker/Rod
	
	if ($@kejc_wBonusLv == 0)	//Don't touch.
		end;

	//Improved versions awarded to second classes
	setarray $@kejc_weapon2_21[0],
		1519,	//Priest/Chain
		1714,	//Hunter/Gakkung
		1610,	//Wizard/Arc Wand
		1360,	//Blacksmith/Two-Handed Axe
		1160,	//Knight/Broad-Sword
		1254,	//Assassin/Jamadhar
		1552;	//Star Gladiator/Tablet

	setarray $@kejc_weapon2_22[0],
		1811,	//Monk/Finger
		1714,	//Bard/Dancer/Gakkung
		1552,	//Sage/Tablet
		1360,	//Alchemist/Two-Handed Axe
		1119,	//Crusader/Tsurugi
		1222,	//Rogue/Damascus
		1610;	//Soul Linker/Arc Wand

	end;
	//-------------------------------------------------------------------------------
	//Config for the Job Swapping Module
	//-------------------------------------------------------------------------------
OnLoadJobSwap:
	set $@kejs_disable,0;	//Disable certain jobs from the swapper:
		//1: Super Novice
		//2: Taekwon/Star Gladiator/Soul Linker
		//4: GunSlinger
		//8: Ninja

	set $@kejs_revertPolicy,1;	//Allows reverting to the last Class one changed from...
		// 0: do not allow reverting
		// 1: Can revert to last Job ONLY when the last job belongs to the same "level" as 
		//    the current one. (ie: it is lost on job change/upgrade)
		// 2: Can revert to the last job regardless of situation.

	set $@kejs_announce,0;	//Do a global announce when someone swaps job.
	set $@kejs_revertCost,100000;	//Cost of reverting back to previous Job.
	set $@kejs_saveDye,1;	//Set to 1 to save and restore previous dye when reverting
	set $@kejs_resetDye,0;	//Set to 1 to reset cloth dye when swapping
	set $@kejs_swapDiscount, 20;	//Kafra Pass % Discount when swapping jobs
	set $@kejs_revertDiscount, 20;	//Kafra Pass % Discount when reverting to a previous job.
	
	//Minimum Job level required for swapping
	set $@kejs_job1ST,20;	//among 1st classes
	set $@kejs_job2ND,50;	//among 2nd classes
	
	//Base Cost for swapping job at min job level required
	set $@kejs_cost1ST,25000;	//among 1st jobs
	set $@kejs_cost2ND,250000;	//among 2nd jobs

	//Percentage discount for every level over the minimum
	set $@kejs_discount1ST,3;	//among 1st jobs
	set $@kejs_discount2ND,3;	//among 2nd jobs
	
	//How much to preserve of the previous Job in %?
	//(25% means that if you change at Lv100, you will become a Lv25 upon swap)
	set $@kejs_preserve1ST,50;	//among 1st jobs
	set $@kejs_preserve2ND,0;	//among 2nd jobs
	end;
	//-------------------------------------------------------------------------------
	//Config for the Refining Module
	//-------------------------------------------------------------------------------
OnLoadRefine:
	set $@kerf_maxLv,10;	//Max refinement level for equipment
	set $@kerf_safe,0;	//Set to 1 to enable 100% safe refinement.
	set $@kerf_showChance,0;	//Set to 1 to see the refine Success Chance
	set $@kerf_refineAll,0;	//Set to 1 to enable refinement of anything (ie: accesories)
	set $@kerf_discount,0;	//Discount % rate for forging when Kafra Pass is active.
	set $@kerf_armorCost,2000;	//Cost to refine armor (default 2000z)
	set $@kerf_weaponLv1Cost,50;	//Cost to refine lv1 weapons (default 50z)
	set $@kerf_weaponLv2Cost,200;	//Cost to refine lv2 weapons (default 200z)
	set $@kerf_weaponLv3Cost,5000;	//Cost to refine lv3 weapons (default 5000z)
	set $@kerf_weaponLv4Cost,20000;	//Cost to refine lv4 weapons (default 20000z)
	set $@kerf_purifyAll,0;	//Set to 1 to enable purifying elemental stones and the like
	set $@kerf_repairCost,10000;	//Cost to repair broken equipment.
	set $@kerf_repairDiscount,25;	//Discount % rate for repairing when Kafra Pass is active
	set $@kerf_repairSteel,1;	//Set to 1 to make repairing equipment require 1 Steel.
	end;
	//-------------------------------------------------------------------------------
	//Config for the Renting Module
	//-------------------------------------------------------------------------------	
OnLoadRent:
	set $@kert_cartOnly, 0;	//Set to 1 to enable only cart rental, 0 enables all add-ons.
	set $@kert_cartCost, 1000;	//Cost to rent a Cart.
	set $@kert_falconCost, 2000;	//Cost to rent a Falcon.
	set $@kert_pecoCost, 2000;	//Cost to rent a PecoPeco.
	end;
	//-------------------------------------------------------------------------------
	//Config for the Kafra Shop Module
	//-------------------------------------------------------------------------------
OnLoadShop:
	set $@kesh_towntype, 0; //Town shop type (1: buy, 2: sell, Other: both)
	set $@kesh_duntype, 2; //Dungeon shop type (1: buy, 2: sell, Other: both)
	set $@kesh_townshop$, "ke_townshop"; //Name of town shop to invoke.
	set $@kesh_dunshop$, "ke_dunshop"; //Name of dungeon shop to invoke.
	end;
	//-------------------------------------------------------------------------------
	//Config for the Stat Market Module
	//-------------------------------------------------------------------------------	
OnLoadStatMarket:
	set $@kesm_stBuyPrice,10000;	//Cost of buying stat points
	set $@kesm_stSellPrice,0;	//Return cash for selling stat points
	set $@kesm_skBuyPrice,1000000;	//Cost of buying skill points
	set $@kesm_skSellPrice,50000;	//Return cash for selling skill points
	set $@kesm_skTradePrice, 20; //Stats needed to trade for a skill.	(Stat->Skill)
	set $@kesm_stTradePrice, 15; //Stats received per Skill traded. (Skill->Stat)
	set $@kesm_discount,0;	//Discount % applied when buying if Kafra Pass is active
	end;
	//-------------------------------------------------------------------------------
	//Config for the Stats Reset/Raising Module
	//-------------------------------------------------------------------------------
OnLoadStats:
	set $@kest_stResetCost,50000;	//Cost to reset stats
	set $@kest_skResetCost,50000;	//Cost to reset skills
	set $@kest_resetCost,75000;	//Cost to reset both
	set $@kest_BaseLvCost,25000;	//Additional cost per base level to reset stats
	set $@kest_JobLvCost,25000;		//Additional cost per job level to reset skills
	set $@kest_BothLvCost,20000;	//Additional cost per base+job level to reset both

	set $@kest_discount,15;	//Discount % to be applied when Kafra Pass is active.
	set $@kest_resetBasic,0;	//Set to 1 if you want basic skills resetted as well.
	end;
	//-------------------------------------------------------------------------------
	//Config for the Uncarding Module
	//-------------------------------------------------------------------------------
OnLoadUncard:
	set $@keuc_BaseCost, 100000;	//Base Cost
	set $@keuc_CardCost, 25000;	//Extra Cost per Card
	set $@keuc_UpgradeCostA, 1000;	//Extra Cost per Upgrade, Armor
	set $@keuc_UpgradeCostW1, 1000;	//Extra Cost per Upgrade, Weapons Lv1
	set $@keuc_UpgradeCostW2, 2000;	//Extra Cost per Upgrade, Weapons Lv2
	set $@keuc_UpgradeCostW3, 3000;	//Extra Cost per Upgrade, Weapons Lv3
	set $@keuc_UpgradeCostW4, 5000;	//Extra Cost per Upgrade, Weapons Lv4
	set $@keuc_discount, 10;	//% Discount when using a Kafra Pass
	set $@keuc_Mat1,1000;	//Material #1 needed (Star Crumb)
	set $@keuc_Qty1,1;		//Qty of material #1 needed (use 0 to disable)
	set $@keuc_Mat2,715;	//Material #2 needed (Yellow Gemstone)
	set $@keuc_Qty2,1;	//Qty of material #2 needed (use 0 to disable)
	
	//Failure Chances. 1000 = 100.0%, set to 0 to disable failing chance.
	set $@keuc_Fail0Chance, 250;	//Chance of harmless failure
	set $@keuc_Fail1Chance, 100;	//Chance of losing item OR card
	set $@keuc_Fail2Chance, 25;	//Chance of losing both
	end;
	//-------------------------------------------------------------------------------
	//Config for the Town Warps Module
	//-------------------------------------------------------------------------------
OnLoadWarpTown:
	set $@kewt_showOnline,1;	//Set to 1 to show online count on cities
	set $@kewt_cost, 1000;	//Cost of warping to a town
	set $@kewt_niflCost, 5000;	//Cost of warping to Niflheim
	set $@kewt_travel, 0;	//Traveller's Mode type for towns:
		//0: Disabled (normal warping)
		//1: You can only warp to towns you've been before (per character basis)
		//2: You can only warp to towns you've been before (per account basis)
	set $@kewt_free, 0;	//Set to 1 to let people warp to towns even when poor
	set $@kewt_discount, 100;	//Discount % to apply when the Kafra Pass is active (towns)
	end;
	//-------------------------------------------------------------------------------
	//Config for the Dungeons Warps Module
	//-------------------------------------------------------------------------------
OnLoadWarpDungeon:
	set $@kewd_showOnline,0;	//Set to 1 to show online count on dungeons
	set $@kewd_discount, 50;	//Discount % to apply when the Kafra Pass is active
	set $@kewd_travel, 0;	//Traveller's mode type for dungeons:
		//0: Disabled (normal warping)
		//1: You can only warp to dungeons you've been before (per character basis)
		//2: You can only warp to dungeons you've been before (per account basis)
	set $@kewd_deep, 1; //If 1, people can warp to all levels of dungeons.
	                    //If 0, people can only warp to the entrance of dungeons.
	
	//Variables for Deep Warping
	set $@kewd_levelCost, 30;	// cost % increase for warping to floors deep into the dungeon
	set $@kewd_entryDiscount, 25;	//Discount % when using the Kafra right next to the dungeon one is warping into
	
	//Variables for Short Warping
	set $@kewd_turtleCave,0; //If 1, Turtle Island's warp leads to the
	                         //entrance of the cave, not the Island's shore.

	//Dungeon Costs
	set $@kewd_abyssLake,3250;	//Abyss Lake base cost
	set $@kewd_amatsu,2500;	//Amatsu Dungeon base cost
	set $@kewd_antHell,750;	//Anthell base cost
	set $@kewd_ayothaya,2000;	//Ayothaya Dungeon base cost
	set $@kewd_byalan,1000;	//Byalan base cost
	set $@kewd_comodo,1500;	//Comodo Caves base cost
	set $@kewd_clockTower,2250;	//Clock Tower base cost
	set $@kewd_coalMines,1000;	//Coal Mines base cost
	set $@kewd_culvert,0500;	//Prontera Culvert base cost
	set $@kewd_einbech,2250;	//Einbech Mines base cost
	set $@kewd_gefenia,7000;	//Gefenia base cost
	set $@kewd_geffen,1500;	//Geffen Dungeon base cost
	set $@kewd_glastHeim,2500;	//Glast Heim base cost
	set $@kewd_gonRyun,2250;	//Gonryun Dungeon base cost
	set $@kewd_hiddenTemple,750;	//Hidden Temple base cost
	set $@kewd_juperos,3750;	//Juperos base cost
	set $@kewd_kiehl,3400;	//Kiehl Doll Factory base cost
	set $@kewd_iceCave,3750;	//Ice Cave base cost
	set $@kewd_lightHalzen,4500;	//LightHalzen Rekkenber base cost
	set $@kewd_louYang,2500;	//Louyang Dungeon base cost
	set $@kewd_magma,3000;	//Magma Dungeon base cost
	set $@kewd_odin,2750;	//Odin Temple base cost
	set $@kewd_orc,1000;	//Orc Dungeon base cost
	set $@kewd_payon,1000;	//Payon Dungeon base cost
	set $@kewd_pyramid,2000;	//Pyramids base cost
	set $@kewd_rachel,3000;	//Rachel Sanctuary base cost
	set $@kewd_sphinx,1750;	//Sphinx base cost
	set $@kewd_sunkenShip,1250;	//Sunken Ship base cost
	set $@kewd_thanatosTower,3250;	//Thanatos Tower base cost
	set $@kewd_toyFactory,1250;	//Toy Factory base cost
	set $@kewd_turtleIsland,3250;	//Turtle Island base cost
	set $@kewd_umbala,1750;	//Umbala Dungeon base cost
	end;
	//-------------------------------------------------------------------------------
	//Config for the Pvp Warps Module
	//-------------------------------------------------------------------------------
OnLoadWarpPvp:
	set $@kewp_showOnline,1;	//Set to 1 to show online count on pvp maps
	set $@kewp_advanced,0; //Set to 0 to enable a single group of pvp rooms
	                       //(no level restrictions, kewp_cost is used)
	set $@kewp_cost, 1000; //Cost for Free For All (no level restrictions)
	
	if ($@kewp_advanced == 0)	//Don't touch
		end;
	
	set $@kewp_cost1, 100; //Cost of warping to a pvp arena, lvls group 1
	set $@kewp_cost2, 250; //Cost for lvls group 2
	set $@kewp_cost3, 500; //Cost for lvls group 3
	set $@kewp_cost4, 1000; //Cost for lvls group 4
	set $@kewp_cost5, 2000; //Cost for lvls group 5
	set $@kewp_cost6, 4000; //Cost for lvls group 6
	set $@kewp_cost7, 8000; //Cost for lvls group 7
	set $@kewp_baseLv1, 35; //Level required for pvp, lv1
	set $@kewp_baseLv2, 45; //Level required for pvp, lv2
	set $@kewp_baseLv3, 55; //Level required for pvp, lv3
	set $@kewp_baseLv4, 65; //Level required for pvp, lv4
	set $@kewp_baseLv5, 75; //Level required for pvp, lv5
	set $@kewp_baseLv6, 85; //Level required for pvp, lv6
	set $@kewp_baseLv7, 95; //Level required for pvp, lv7
	set $@kewp_range, 5; //Range of levels allowed (ie: Warp to Lv1: levels 10+/-5)
	end;
	//-------------------------------------------------------------------------------
	//Config for the WoE Warps Module
	//-------------------------------------------------------------------------------
OnLoadWarpWoe:
	//Note: See ke_warp_woe.txt for instructions on how to unlock the Guild Dungeons.
	set $@kewg_check,1;	//If 0, people don't need to belong to a Guild to warp
	set $@kewg_checkAgit,1;	//If 1, you can't warp until WoE starts
	set $@kewg_showOnline,0;	//Set to 1 to show online count and Castle Status
	set $@kewg_discount,20;	//Kafra Pass discount for WoE warps

	set $@kewg_baldur, 25000;	//Guild Dungeon - Baldur
	set $@kewg_luina, 25000;	//Guild Dungeon - Luina
	set $@kewg_valkyrie, 25000;	//Guild Dungeon - Valkyrie
	set $@kewg_britoniah, 25000;	//Guild Dungeon - Britoniah

	set $@kewg_alDeBaran, 2500;	//Al De Baran Guild
	set $@kewg_geffen, 2500;	//Geffen Guild
	set $@kewg_payon, 2500;	//Payon Guild
	set $@kewg_prontera, 2500;	//Prontera Guild
	end;
	//-------------------------------------------------------------------------------
}


CPanel/CHANGELOG.txt
Date

2008/05/17
	* Changed password.php and changemail.php to POST method. [BeoWulf]
	* Fixed the mail sending function, now the recover password function should work, also
	now that we're using the phpmailer class you should be able to use gmail smtp server too [Dekamaster]
	* Disabled error report by default. [Dekamaster]
	* Added a definition file for the e-mail template, now you can customize the template of
	the email that will be sent to your players in /language/mail/ [Dekamaster]
	* Added Castle names, kisuka report on eAthena forums. [BeoWulf]
	* Added extra protection to login.php after some security leak rumors (not yet confirmed though). [BeoWulf]
	* Testing a new Captcha, its uglier than the original, but more safe. [Dekamaster]
	* Extended some of login.php protection to account.php and recover.php just in case. [BeoWulf]

2008/05/14
	* Fixed a bug with reset position leting a jailed person to get out, reported by panchox
	at ea boards (and thanks for the temporary fix, brianlew, we really appreciated it.) [Dekamaster]

------------------
We are retaking the project today (2008/05/14), we'll be slowly adding new stuffs to the control
panel and fixing old bugs, if you have a bug to report you are welcome to share it with us at
cerescp@gmail.com (yes we are reading it again)
------------------

2007/01/01
	* Fixed Firefox incompatibility, stupid browser. [BeoWulf]

2006/12/03
	* Reviewed img.php and middle.php files with help of gunawan wibisono. [BeoWulf]

2006/10/23
	* Added a variable in recover.php. [BeoWulf]

2006/10/18
	* Fixed an exploit in language.php. [BeoWulf]

2006/10/15
	* Added a one day ipban to users blocked, it will keep then out to think about
	what they did and can't creat accounts with ofensive nicks against GMs. [BeoWulf]
	* Small fix in server_status.php, and a change in account.php so now its
	post not get (internal stuff). [BeoWulf]

2006/10/03
	* Fixed errors in emblems.php, adminquery.php, ladder.php and functions.php. [BeoWulf]

2006/10/03
	* Changed all session maybe now there is no conflict with 2 cps in same machine. [BeoWulf]

2006/09/22
	* Added some fuzzy to code image. [BeoWulf]

2006/09/18
	* Created first trial of an install script. [BeoWulf]
	* Fixed install script checks and w3c instructions. [BeoWulf]
	* Changed INSTALL.TXT instructions to fit the new install script. [BeoWulf]

2006/09/18
	* Added jail check to position.php so jailed characters may not reset
		their position. [Entwined]
	* Added Post functions do ajax engine. [BeoWulf]
	* Changed login.php from get to post. [BeoWulf]
	* Fixed whoisonline.php adjust to show position to minimum admin level set 
		in config.php. [BeoWulf]
	* Added check in functions.php to destroy session if you use more than one
		CP in same machine when changing beteween them use different names 
		for the CPs or it can be exploitable, still working on a different 
		solution. [BeoWulf]
	* Changed admincharinfo.php to a beta test in changing stored char_id in 
		equip slots into something we can read. [BeoWulf]
	* Added adminchars.php still working on it, to manage chars. [BeoWulf]
	* Added money.php testing a 2 minutes ban after transfer. [BeoWulf]
	* Added marriage.php testing a 2 minutes ban after divorce. [BeoWulf]
	* Added guild.php if castle name is not defined it will skip and not show. [BeoWulf]
	* Added dutch and indonesian to languages. [BeoWulf]
	* Fixed some stuff in other language files, but still need translations. [BeoWulf]
	* Fixed ladder.php and about.php to show the new classes. [BeoWulf]

2006/09/14
	* Added map and gm level check for position reset, so you can't reset your position if your in jail [Entwined]
	* Added install/sql/rag_ragsrvinfo.sql and install/npc/agit_status.txt. You will have to install both of 
		these correctly for agit status	check to work. [Entwined]
	* Added ceres.php for control panel information [Entwined]
	* Added max account check for account registration [Entwined]

2006/09/13
	* Added IP address field insertion on registrations (last_ip) [Entwined]
	* Added an email verification field for account registration. [Entwined]
	* Added some config settings for future use. [Entwined]
	* Created the initial repository of CeresCP (rev4) as the working base. [Entwined]


CPanel/config.php
<?php
/*
Ceres Control Panel

This is a control pannel program for Athena and Freya
Copyright (C) 2005 by Beowulf and Nightroad

This program is free software; you can redistribute it and/or
modify it under the terms of the GNU General Public License
as published by the Free Software Foundation; either version 2
of the License, or (at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.

To contact any of the authors about special permissions send
a mail to cerescp@gmail.com
*/

//sql connections
$CONFIG['db_serv']		=	'216.180.250.123';	// SQL Host
$CONFIG['db_user']		=	'cerescp';		// SQL User
$CONFIG['db_pass']		=	'f?IBn],~CuPB';		// SQL Password
$CONFIG['rag_db']		=	'ragnarok';		// SQL Ragnarok Database name
$CONFIG['cp_db']		=	'ragnarok';			// SQL CP Database name
$CONFIG['log_db']		=	'ragnarok';		// SQL Ragnarok Log Database name
$CONFIG['md5_pass']		=	'0';			// Use MD5 password (enable = 1, disable = 0)
$CONFIG['safe_pass']		=	'1';			// Force the use of a safer password with size 6 and at least 2 letter and 2 numbers (enable = 1, disable = 0)

//Admin Area
$CONFIG['cp_admin']		=	'99';			// CP admin functions
$CONFIG['gm_level']		=	'100';			// CP GM funtions

//WOE
// sun = sunday, mon = monday, tue = tuesday, wed = wednesday, thu = thursday, fri = friday, sun = sunday
// place week_day(start_time, end_time) and a ';' between the times the freya default woe times is set as an example
// there is no limit you can place as many as you want, no spaces are needed, but using it you can understand.
$CONFIG['woe_time']		=	'wed(1800, 1900); thu(1800, 1900); sat(1400, 1600); sun(1400, 1600); ';
$CONFIG['agit_check']		=	'0';			// This WILL NOT WORK unless you installed the npc script AND you updated your ragsrvinfo table, read the installation notes for more info.

//server name, rates
$CONFIG['name']			=	'::Brawl Network::';	// name of the server
$CONFIG['rate']			=	'500/500/200';		// rates of the server
date_default_timezone_set('America/El_Salvador');		// game server Timezone (useful if your webserver's timezone is different than game server).
$CONFIG['dynamic_info']		=	'0';			// Use info (rates) from the server itself?
$CONFIG['dynamic_name']		=	'Brawl Network';	// The name of the server in ragsrvinfo's server name column (Used for dynamic info)
$CONFIG['show_rates']		=	'0';			// Show rates below server status?

//map,char,login servers settings
$CONFIG['accip']		=	'216.180.250.123';	// Account/Login Server IP
$CONFIG['accport']		=	'6900';		// Account/Login Server Port
$CONFIG['charip']		=	'216.180.250.123';	// Char Server IP
$CONFIG['charport']		=	'6121';		// Char Server Port
$CONFIG['mapip']		=	'216.180.250.123';	// Zone/Map Server IP
$CONFIG['mapport']		=	'5121';		// Zone/Map Server Port

//default language
$CONFIG['language']		=	'Espanol';		// default language (remember to check if the translation exist before set)

//cp features
$CONFIG['disable_account']	=	'0';			// disable the account creation disable = 1, enable = 0
$CONFIG['auth_image']		=	'1';			// enable the verification code image, to check if it's a real person using the cp, instead of a bot (brute-force atack) - Recommended, but requires gd library (enable = 1 disable = 0)
$CONFIG['max_accounts']		=	'0';			// Max accounts allowed to be in the DB (0 = disabled)
$CONFIG['password_recover']	=	'1';			// password recover enable = 1, disable = 0
$CONFIG['reset_enable']		=	'1';			// reset position enable = 1, disable = 0
$CONFIG['reset_cost']		=	'300';		// reset position cost, disable cost = 0
$CONFIG['money_transfer']	=	'0';			// money transfer enable = 1, disable = 0
$CONFIG['money_cost']		=	'0';			// money transfer cost (100 = 1%), disable cost = 0
$CONFIG['set_slot']		=	'1';			// change char slot enable = 1, disable = 0
$CONFIG['reset_look']		=	'1';			// reset char equips and colors with error enable = 1, disable = 0
$CONFIG['marry_enable']		=	'1';			// enable marriage view and divorce
$CONFIG['prison_map']		=	'sec_pri';		// Name of the map that is used as your jail (mapname.gat)

//About Information
$CONFIG['classlist_show']	=	'1';			// Show the class list on about.php? (disable = 0, enable = 1)

//Mail
$CONFIG['smtp_server']		=	'server405.webhostingpad.com';	// the smtp server, the cp will use to send mails
$CONFIG['smtp_port']		=	'2626';			// the smtp server port
$CONFIG['smtp_mail']		=	'cuentas@brawlnetwork.net';		// the email of the admin
$CONFIG['smtp_username']	=	'cuentas@brawlnetwork.net';			// the username of the smtp server
$CONFIG['smtp_password']	=	'PayRcwTfTt-y';			// the password of the smtp server

//define('VOTE_TIME', 24 );
//define('VOTE_LINK', serialize( array(
                                        //1 => 'http://top.ragnarokonline.es/index.php?a=in&u=Siver?id=1',
                                        //2 => 'http://top.ragnarokonline.es/index.php?a=in&u=Siver?id=2',
                                        //3 => 'http://www.voteseite3.com?id=123456'
                            //))
        //);
//DO NOT MESS WITH THIS
extract($CONFIG, EXTR_PREFIX_ALL, "CONFIG");
extract($_GET, EXTR_PREFIX_ALL, "GET");
extract($_POST, EXTR_PREFIX_ALL, "POST");
extract($_SERVER, EXTR_PREFIX_ALL, "SERVER");
error_reporting(0);
?>